package.json

```diff
@@ -18,23 +20,22 @@
     "loglevel": "^1.8.0",
     "prop-types": "^15.8.1",
     "query-string": "^7.1.1",
-    "react": "^16.14.0",
+    "react": "17.0.2",
     "react-app-polyfill": "^3.0.0",
-    "react-dom": "^16.14.0",
+    "react-dom": "17.0.2",
     "react-ga": "^3.3.0",
     "react-i18next": "^11.15.3",
     "react-joyride": "^2.3.2",
-    "react-redux": "^7.2.0",
-    "react-redux-toastr": "^7.6.5",
-    "react-router": "^5.0.1",
+    "react-redux": "^7.2.5",
+    "react-redux-toastr": "^7.6.8",
     "react-router-dom": "^5.3.0",
     "react-scripts": "5.0.0",
     "redux": "^4.1.2",
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.4.1",
     "single-spa": "^5.9.1",
     "typeface-roboto": "1.1.13",
-    "typescript": "4.6.2"
+    "typescript": "4.5.5"
```

Comment:
Is this typescript downgrade intentional?

Reply:
It seems like there's a few downgrades in here

Reply:
Yes this upgrade was a recent change with eslint which I had to downgrade again because two of the packages were conflicting, the typescript version also had to go back as the new version is unsupported by the older eslint and so gives a warning.

Reply:
Okay - can you create an issue to track this? As otherwise it will likely be forgotten

Reply:
@louise-davies Wouldn't dependabot just pick this up again?

Reply:
I'll make an issue

Reply:
@sam-glendenning Just to clarify it appeared after
https://github.com/ral-facilities/scigateway/pull/969
https://github.com/ral-facilities/scigateway/pull/973
https://github.com/ral-facilities/scigateway/pull/971
https://github.com/ral-facilities/scigateway/pull/968
were merged and seemed to be caused by eslint-config-prettier requiring a different version of @typescript-eslint/parser.